### PR TITLE
Decode 1inch v4 transactions using OrderFilledRFQ

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

File: rotkehlchen/tests/unit/decoders/test_1inch.py [hunk 1]
```diff
@@ -297,6 +297,60 @@ def test_1inchv4_swap_on_uniswapv3(database, ethereum_inquirer):
     assert expected_events == events
 
 
+@pytest.mark.vcr()
```
Comment (yabirgb) [inline rotkehlchen/tests/unit/decoders/test_1inch.py:300]: you didn't add the api key filter here

```suggestion
@pytest.mark.vcr(filter_query_parameters=['apikey'])
```

I don't recall where is required but I belive the test will run fine

Comment (LefterisJP) [reply rotkehlchen/tests/unit/decoders/test_1inch.py:300]: it will run fine

Comment (LefterisJP) [reply rotkehlchen/tests/unit/decoders/test_1inch.py:300]: or maybe not :rofl:

Comment (LefterisJP) [reply rotkehlchen/tests/unit/decoders/test_1inch.py:300]: hmmmm it's unrelated. I think I did not push the cassettes PR ffs

**Comment by LefterisJP** [comment]
all green now

**Pull request merged**